tests/unit/test_dataset_geo.py

```diff
@@ -513,6 +540,57 @@ def __geo_interface__(self):
         actual = [json.loads(item) for item in snoop["payload"]]
         self.assertEqual(expected, actual)
 
+    @responses.activate
+    def test_from_geo_features_geo_attr(self):
+        def update_callback(request, snoop):
+            snoop["payload"] = request.body
+            return 200, {}, "{}"
+
+        dataset_url = f"http://localhost:9100/api/versioned/v1/datasets/1"
+        responses.add(responses.GET, dataset_url, json=self._dataset_json)
+
+        # Create a dataset with multiple geometry attributes
```

Comment:
why is there the same thing twice? (see line 266)

Reply:
this is about line 552 and the ensuing chunk of code

Reply:
The first is testing `itergeofeatures`, the second is testing `from_geo_features`